home/templates/pages/search-results.html

```diff
@@ -51,6 +51,7 @@
 
   {% block content %}
     <div class="container">
+    <div id="loading-indicator" style="display:none;">Loading...</div>
```

Comment:
So when this is visible, which currently it never is, its at the top of the page under the navigation bar
could you reposition it so when it is visible its correctly positions

Reply:
ill fix it, thank you